components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -158,28 +157,24 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
               classNames="bg-light-slate-4 text-light-slate-11 font-medium"
               placeholder="https://opensauced.pizza"
               label="URL"
-              disabled
             />
             <TextInput
               classNames="bg-light-slate-4 text-light-slate-11"
               placeholder="@saucedopen"
               label="Twitter Username"
-              disabled
-              value={`@${(userInfo && userInfo.twitter_username) || "saucedopen"}`}
+              defaultValue={`@${(userInfo && userInfo.twitter_username) || "saucedopen"}`}
```

Comment:
Why the switch from `value` to `defaultValue`? We'll also need API updates to enable these additional fields to be updated

Reply:
Setting `value` blocks the user from changing the value of these fields. 
`defaultValue` only sets the starting value of the field and allows the user to change it.

Reply:
Yes, but its not connected to the form, so it won't get captured with the form submission. We need to either

- Create a `useState` for each field to track the changes. Pass the state as `value` and update it when the input changes.
- Create a `useReducer` to track the overall state and update it for each field in the form. Then use the value from the state to submit with the form.

I would recommend the second option

Reply:
We could also use `useRef` to get the value of each input at form submission.

Reply:
I like that option also